pkg/views/profile/create.go

```diff
@@ -29,8 +29,8 @@ func ProfileCreationView(c *config.Config, profileAddView *ProfileAddView, editi
 			if str == "" {
 				return errors.New("profile name can not be blank")
 			}
-			if match, _ := regexp.MatchString("^[a-zA-Z0-9]+$", str); !match {
-				return errors.New("profile name must be alphanumeric only")
+			if match, _ := regexp.MatchString("^[a-zA-Z0-9._-]+$", str); !match {
+				return errors.New("invalid profile name. Only [a-zA-Z0-9._-] are allowed")
```

Comment:
A more user-friendly message would be appreciated. E.g. `Only letters, digits, dashes and periods are allowed`